src/NzbDrone.Core/Notifications/Mailgun/MailgunSettings.cs

```diff
@@ -0,0 +1,54 @@
+using System.Collections.Generic;
+using System.Net;
+using System.Runtime.CompilerServices;
+using FluentValidation;
+using NzbDrone.Core.Annotations;
+using NzbDrone.Core.ThingiProvider;
+using NzbDrone.Core.Validation;
+
+namespace NzbDrone.Core.Notifications.MailGun
+{
+    public class MailGunSettingsValidator : AbstractValidator<MailGunSettings>
+    {
+        public MailGunSettingsValidator()
+        {
+            RuleFor(c => c.ApiKey).NotEmpty();
+            RuleFor(c => c.From).NotEmpty();
+            RuleFor(c => c.Recipients).NotEmpty();
+        }
+    }
+    
+    public class MailGunSettings : IProviderConfig
+    {
+      private static readonly  MailGunSettingsValidator Validator = new MailGunSettingsValidator();
+
+      public MailGunSettings()
+      {
+          Recipients = new string[] { };
+      }
+      
+      
+      [FieldDefinition(1, Label = "API Key", HelpText = "The API key generated from MailGun")]
+      public string ApiKey { get; set; }
+      
+      [FieldDefinition(2, Label = "Use EU Endpoint?", HelpText = "You can choose to use the EU MailGun endpoint with this", Type = FieldType.Checkbox)]
+      public bool UseEuEndpoint { get; set; }
+      
+      [FieldDefinition(3, Label = "From Address")]
+      public string From { get; set; }
+      
+      [FieldDefinition(4, Label = "Sender Domain")]
+      public string SenderDomain { get; set; }
+      
+      [FieldDefinition(5, Label = "Recipient Address(es)", Type = FieldType.Tag)]
+      public IEnumerable<string> Recipients { get; set; }
+
+      [FieldDefinition(9, Label = "Test mode", HelpText = "Only test to send message, but do not send it", Advanced = true, Type = FieldType.Checkbox)]
```

Comment:
The `HelpText` doesn't really tell me what this option will do. Why would I want to enable this?